### PR TITLE
Replaced all the css into tailwind css on TimelineNav component

### PR DESCRIPTION
**Pull request by subhajit20**
# Description of changes

Replaced all the css into tailwind css on TimelineNav component
Issue - #1725 

# Issue Resolved

Fixes #NA

### CODE REVIEW

File: components/Timeline/TimelineNav/TimelineNav.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
 import { Link } from 'react-scroll';
 import historyData from '../historyData';
-import styles from './TimelineNav.module.css';
 
 const TimelineNav = () => {
   return (
-    <section className={styles.timelineNav}>
-      <nav className={styles.timelineNavLinksContainer}>
+    <section className="py-0 px-16 sm:p-0">
```
Comment (kylemh) [inline components/Timeline/TimelineNav/TimelineNav.js:6]:
```suggestion
    <section className="px-16 sm:p-0">
```

File: components/Timeline/TimelineNav/TimelineNav.js [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+      <nav className="flex justify-around">
         {Object.keys(historyData).map(year => (
-          <Link key={year} to={`event-${year}`} smooth className={styles.timelineNavLink}>
+          <Link
+            key={year}
+            to={`event-${year}`}
+            smooth
+            className="font-bold text-xl cursor-pointer transition-all duration-100 ease-in-out hover:scale-[1.1]"
```
Comment (kylemh) [inline components/Timeline/TimelineNav/TimelineNav.js:13]:
```suggestion
            className="font-bold text-xl cursor-pointer opacity-60 transition-all duration-100 ease-in-out hover:opacity-100 hover:scale-[1.1]"
```

File: components/Timeline/TimelineNav/TimelineNav.js [hunk 3]
```diff
@@ -1,18 +1,22 @@
...
+          >
             {year}
           </Link>
         ))}
       </nav>
-      <div className={styles.timelineNavSeperator} />
+      <div className="mt-4 bg-[#3d7d7d7] h-1" />
```
Comment (kylemh) [inline components/Timeline/TimelineNav/TimelineNav.js:19]:
```suggestion
      <div className="mt-4 bg-[#d7d7d7] h-1" />
```

**Comment by subhajit20** [comment]
Hi @kylemh ,
Is there any changes needed?

**Comment by subhajit20** [comment]
Btw Happy New Year 🙏🏻

**Review by kylemh — APPROVED** [review]
happy new year!

**Pull request merged**